### PR TITLE
Update publish-examples.md

### PR DESCRIPTION
**Pull request by organisergirl**

### CODE REVIEW

**Review by hot007 — APPROVED** [review]
Nice work @organisergirl , this is a good clear re-write, thank you!
I've suggested a bunch of changes, hopefully they're pretty minor and they're just suggestions, it's fine to merge as is (pending others' reviews I guess).

File: Governance/publish/publish-examples.md [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
+
+- Dix, Martin; Bi, Dave; Dobrohotoff, Peter; et. al (2019): ACCESS-CM2 model output prepared for CMIP6 under 'ScenarioMIP'. v2. CSIRO. Service Collection. http://hdl.handle.net/102.100.100/422393?index=1 
+
+- Durrant, Thomas; Hemer, Mark; Smith, Grant; Trenham, Claire; Greenslade, Diana (2019): CAWCR Wave Hindcast - Aggregated Collection. v5. CSIRO. Service Collection. http://hdl.handle.net/102.100.100/137152?index=1
+
+- Earth System Grid Federation (ESGF) Australian CMIP6-era Datasets (2021)
+NCI. http://doi.org/10.25914/5e6acd0492b39
+
+
+[^1]: Trenham, Claire; Steer, Adam. The Good Data Manifesto. In: Daly, Angela; Devitt, Kate; Mann, Monique, editor/s. Good Data. Amsterdam: Institute of Network Cultures (INC) Publications; 2019. 37-53. https://networkcultures.org/blog/publication/tod-29-good-data/.
```
[the remark this answers is not in the document]
Comment (organisergirl) [reply Governance/publish/publish-examples.md:25]: URLs are resolvable without needing to add more, that's only for when you want to make a word or phrase a link.

File: Governance/publish/publish-examples.md [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
 
-Seasonal Prediction ACCESS-S2 Reanalysis (1981-2018) - http://dx.doi.org/10.25914/60627ad8878e2
+What does best practice look like? Certainly, it DOES look like including all metadata, ensuring data is [FAIR](https://www.go-fair.org/fair-principles/)(ER?) [^1], documenting data processing and publishing that as well, undertaking QA/QC and documenting the process.
 
-AGCD - http://dx.doi.org/10.25914/6009600786063
+It is near impossible to get all this right and even with strict standards like CMIP/CORDEX we still find mistakes in the metadata due to the complexity of the standards. I agree that we will never get this perfect but aim for a best effort and as our tooling (software/workflows) improve over time so will our compliance.
```
[the remark this answers is not in the document]
Comment (organisergirl) [reply Governance/publish/publish-examples.md:7]: Done

File: Governance/publish/publish-examples.md [hunk 3]
```diff
@@ -1,15 +1,25 @@
 # Best Practice Published Data Records
 
-BoM data published at NCI  
+The climate community is in better position with respect to best practice data management than many other groups due to the self-describing data format of [NetCDF](https://www.unidata.ucar.edu/software/netcdf/) and community modelling projects such as [CMIP](https://www.wcrp-climate.org/wgcm-cmip)/[CORDEX]( https://cordex.org/)/[Obs4MIPS]( https://pcmdi.github.io/obs4MIPs/) enforcing a strict standard. There are lots of climate-tools in wide use across the community that make use of this fact, including modifying the history attribute to document what was done to the data.
 
-Seasonal Prediction ACCESS-S2 Reanalysis (1981-2018) - http://dx.doi.org/10.25914/60627ad8878e2
+What does best practice look like? Certainly, it DOES look like including all metadata, ensuring data is [FAIR](https://www.go-fair.org/fair-principles/)(ER?) [^1], documenting data processing and publishing that as well, undertaking QA/QC and documenting the process.
```
[the remark this answers is not in the document]
Comment (organisergirl) [reply Governance/publish/publish-examples.md:5]: Good points.

File: Governance/publish/publish-examples.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (hot007) [reply Governance/publish/publish-examples.md:25]: Hah! I just learned a thing! Thank you :)

**Comment by paolap** [comment]
As Claire I think we should change the link to the FAIRER page in the concepts as it provides a summary and "downscaling" to our kind data. Apart from that I think we need some more down to earth examples, what we're listing there so far is really big dataset, which have set standards anyway. We should show some smaller ones. I'll try to think of some good example

**Comment by hot007** [comment]
My wave data example there (Durrant et al) doesn't follow standards (sadly) as it's a more bespoke style collection - indeed thanks to changes in the model code between v3 and v4 the variables names aren't even consistent throughout the dataset!! :D It is a large dataset (many TB) but it is more akin to regular research output than something well structured like ACCESS.

**Pull request merged**